NitroxPatcher/Patches/Dynamic/Flare_Update_Patch.cs

```diff
@@ -0,0 +1,36 @@
+using System.Collections.Generic;
+using System.Reflection;
+using System.Reflection.Emit;
+using HarmonyLib;
+using NitroxClient.GameLogic;
+using NitroxModel.Helper;
+using UnityEngine;
+
+namespace NitroxPatcher.Patches.Dynamic;
+
+/// <summary>
+/// Replaces local use of <see cref="Time.deltaTime"/> by <see cref="TimeManager.DeltaTime"/>
+/// </summary>
+public sealed partial class Flare_Update_Patch : NitroxPatch, IDynamicPatch
+{
+    public static readonly MethodInfo TARGET_METHOD = Reflect.Method((Flare t) => t.Update());
+    private static readonly MethodInfo INSERTED_METHOD = Reflect.Method(() => GetDeltaTime());
+
+    public static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstruction> instructions)
+    {
+        return new CodeMatcher(instructions).MatchStartForward(
+                                                new CodeMatch(OpCodes.Ldarg_0),
```

Comment:
Is there a reason you can't Match by `Time.deltaTime`? Right now it's a bit difficult to understand what it replaces.